src/scene_server/topo_server/service/inst_business.go

```diff
@@ -458,8 +455,8 @@ func (s *Service) SearchBusiness(ctx *rest.Contexts) {
 		authorizedResources, err := s.AuthManager.Authorizer.ListAuthorizedResources(ctx.Kit.Ctx, ctx.Kit.Header,
 			authInput)
 		if err != nil {
-			blog.Errorf("[api-biz] SearchBusiness failed, ListAuthorizedResources failed, user: %s, err: %s, "+
-				"rid: %s", ctx.Kit.User, err.Error(), ctx.Kit.Rid)
+			blog.Errorf("SearchBusiness failed, list authorized resources failed, user: %s, err: %v, rid: %s",
```

Comment:
这种函数名的日志处理一下，不要这样用，然后如果第一句描述整个函数，后面才是当前操作的要去掉第一句
```suggestion
			blog.Errorf("list authorized resources failed, user: %s, err: %v, rid: %s",
```